Source/ThirdParty/libwebrtc/Source/webrtc/common_video/h265/h265_pps_parser.h

```diff
@@ -29,12 +29,16 @@ class H265PpsParser {
   struct PpsState {
     PpsState() = default;
 
-    bool bottom_field_pic_order_in_frame_present_flag = false;
-    bool weighted_pred_flag = false;
-    bool entropy_coding_mode_flag = false;
-    uint32_t weighted_bipred_idc = false;
-    uint32_t redundant_pic_cnt_present_flag = 0;
-    int pic_init_qp_minus26 = 0;
+    bool dependent_slice_segments_enabled_flag = 0;
+    bool cabac_init_present_flag = 0;
+    bool output_flag_present_flag = 0;
```

Comment:
for initializing bool types, it is better to use "true" or "false" instead of "0".